### PR TITLE
Update EUI description for end device creation

### PR DESCRIPTION
**Pull request by KrishnaIyer**
#### Summary

Update EUI description for end device creation. Refs: https://github.com/TheThingsNetwork/lorawan-stack/pull/5035

#### Changes

Update EUI description for end device creation.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [x] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/devices/adding-devices/_index.md [hunk 1]
```diff
@@ -196,11 +196,13 @@ $ ttn-lw-cli end-devices create app1 dev1 \
 
 This will create a LoRaWAN 1.0.3 end device `dev1` in application `app1` with the `EU_863_870` frequency plan. Please note that the `AppEUI` is returned as `join_eui` ({{% tts %}} uses LoRaWAN 1.1 terminology).
 
-{{< note >}} If you do not have a `JoinEUI` or `AppEUI`, you could use `0000000000000000`. 
+{{< new-in-version "3.17.0">}} If your device does not have a JoinEUI, you can use the default Join EUI of the Join Server. To use this, leave the `--join-eui`/`--app-eui` field empty. The CLI will contact the Join Server and get it's default.
```
Comment (nejraselimovic) [inline doc/content/devices/adding-devices/_index.md:199]:
```suggestion
{{< new-in-version "3.17.0">}} If your device does not have a JoinEUI, you can use the default Join EUI of the Join Server. To use this, leave the `--join-eui`/`--app-eui` field empty. The CLI will contact the Join Server and get its default.
```

File: doc/content/devices/adding-devices/_index.md [hunk 2]
```diff
@@ -196,11 +196,13 @@ $ ttn-lw-cli end-devices create app1 dev1 \
...
 
-Some devices do not support using `0000000000000000` as a JoinEUI/AppEUI, because technically, this value is invalid. However, {{% tts %}} supports using this value to indicate the absence of an actual JoinEUI/AppEUI.
+If your device does not have a DevEUI, set the `--request-dev-eui` flag, which will contact the server and fetch a DevEUI.
```
Comment (nejraselimovic) [inline doc/content/devices/adding-devices/_index.md:201]:
```suggestion
If your device does not have a DevEUI, set the `--request-dev-eui` flag. The CLI will contact the server and fetch a DevEUI.
```

**Review by nejraselimovic — COMMENTED** [review]
This looks good but I suppose this goes for all LoRaWAN versions, not just 1.0.x, right? Because there are other occurrences of this info related to using `0000000000000000` JoinEUI in this doc

**Comment by KrishnaIyer** [comment]
Thanks. I've fixed that.

**Review by htdvisser — APPROVED** [review]

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**